Sources/StringStyle+Part.swift

```diff
@@ -219,81 +219,52 @@ extension StringStyle {
-                        return
-                    case let .headingLevel(headingLevel):
-                        self.headingLevel = headingLevel
-                        return
-                    default:
-                        break
-                    }
-                #endif
-
-                #if os(OSX) || os(iOS) || os(tvOS)
-                    switch stylePart {
-                    case let .numberCase(numberCase):
-                        self.numberCase = numberCase
-                    case let .numberSpacing(numberSpacing):
-                        self.numberSpacing = numberSpacing
-                    case let .fractions(fractions):
-                        self.fractions = fractions
-                    case let .superscript(superscript):
-                        self.superscript = superscript
-                    case let .`subscript`(`subscript`):
-                        self.`subscript` = `subscript`
-                    case let .ordinals(ordinals):
-                        self.ordinals = ordinals
-                    case let .scientificInferiors(scientificInferiors):
-                        self.scientificInferiors = scientificInferiors
-                    case let .smallCaps(smallCaps):
-                        self.smallCaps.insert(smallCaps)
-                    case let .stylisticAlternates(stylisticAlternates):
-                        self.stylisticAlternates.add(other: stylisticAlternates)
-                    case let .contextualAlternates(contextualAlternates):
-                        self.contextualAlternates.add(other: contextualAlternates)
-                    case let .fontFeature(featureProvider):
-                        self.fontFeatureProviders.append(featureProvider)
-                    default:
-                        #if os(iOS) || os(tvOS)
-                            switch stylePart {
-                            case let .adapt(style):
-                                self.adaptations.append(style)
-                            case let .textStyle(textStyle):
-                                self.font = UIFont.bon_preferredFont(forTextStyle: textStyle, compatibleWith: nil)
-                            default:
-                                fatalError("StylePart \(stylePart) should have been caught by an earlier case.")
-                            }
-                        #else
-                            fatalError("StylePart \(stylePart) should have been caught by an earlier case.")
-                        #endif
-                    }
-                #else
-                    fatalError("StylePart \(stylePart) should have been caught by an earlier case.")
-                #endif
-            }
+        case let .hyphenationFactor(hyphenationFactor):
+            self.hyphenationFactor = hyphenationFactor
+        #if os(iOS) || os(tvOS) || os(watchOS)
+        case let .speaksPunctuation(speaksPunctuation):
```

Comment:
I think I'd prefer to keep everything indented inside the `#if` blocks to make it easier to visually chunk this into its constituent parts.

Reply:
Okay, I changed it. I originally removed that indentation because the default Xcode indentation (highlight code, then `control+I`) doesn’t add extra indentation within `#if` blocks.